### PR TITLE
Switch suricata to repeat mode

### PR DESCRIPTION
**Pull request by filippocarletti**
https://github.com/NethServer/dev/issues/6205

The exclusion for 127.0.0.1/32 in shorewall/rules may be required only on non-gateways (i.e. systems having only green zones), more tests needed.

See also NethServer/nethserver-firewall-base#143

### CODE REVIEW

File: root/etc/e-smith/templates/etc/shorewall/rules/15IPS
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/shorewall/rules/15IPS]: Firewall sections must be handled inside firewall template.

Also the exclusion should be more like `127.0.0.0/24`.
Moreover, why not exclude all green IP addresses?

File: root/etc/e-smith/templates/etc/suricata/suricata.yaml/10base
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/suricata/suricata.yaml/10base:1383]: Is this mark good even if ndpi, qos and multiwan are enabled?

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Review by gsanchietti — COMMENTED** [review]
Also make sure to bind this version of `nethserver-suricata` with a specific version of `nethserver-firewall-base` otherwise it will break current installations.

File: root/etc/e-smith/templates/etc/suricata/suricata.yaml/10base
Comment (filippocarletti) [reply root/etc/e-smith/templates/etc/suricata/suricata.yaml/10base:1383]: Yes

**Comment by filippocarletti** [comment]
> Also make sure to bind this version of `nethserver-suricata` with a specific version of `nethserver-firewall-base` otherwise it will break current installations.

Done:
`+Requires: nethserver-firewall-base > 3.9.2`

**Review by gsanchietti — APPROVED** [review]
Wait for the issue before merge.

**Pull request merged**
